src/sentry/replays/post_process.py

```diff
@@ -138,6 +140,11 @@ def _archived_row(replay_id: str, project_id: int) -> dict[str, Any]:
         "started_at": None,
         "is_archived": True,
```

Comment:
Are all the fields from VALID_FIELD_SET in the dictionary?  If not we should add them.

Reply:
added!